docs/geolocation.md

```diff
@@ -0,0 +1,168 @@
+## Migration Guide: Converting Legacy `map_fields.PointField` to Django GIS `PointField`
+
+This documentation explains how to migrate from the custom and legacy point field (`map_fields.PointField`) to Django’s standard GeoDjango `PointField`, based on the latest changes in the branch and the example from [PR #6023](https://github.com/liqd/a4-meinberlin/pull/6023/files).
+
+So far we have migrated the projects app, for which migration field and data is hapenning in the adhocracy4
+Adhocracy+ apps that need to migrate in the future:
+- mapideas
+- budgeting
+
+---
+
+### 1. Background
+
+In the following example, we outline the steps required for changing the mapideas module.
+
+Previously, the project used a custom `PointField` from `adhocracy4.maps.fields`:
+
+```python
+# Before (legacy field)
+from adhocracy4.maps import fields as map_fields
+
+class AbstractMapIdea(models.Model):
+    point = map_fields.PointField()
+```
+
+This field is now **removed** in favor of Django’s built-in GIS field:
+
+```python
+from django.contrib.gis.db import models as gis_models
+
+class AbstractMapIdea(models.Model):
+    coordinates = gis_models.PointField(srid=4326, null=True, blank=True)
```

Comment:
I guess this is mostly straight-forward, but maybe add an explanation of the SRID? Or since the default ist `4326`, this can be omitted in most cases?

Reply:
indeed can be omitted